.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/sarthak/gsoc/aboutcode-toolkit/local/bin/python2.7"
```

Comment:
Perhaps this should be removed and shouldn't be committed

Reply:
I deleted the folder, I don't know why is this still showing up.